python/ray/exceptions.py

```diff
@@ -367,6 +367,33 @@ def __str__(self):
         )
 
 
+@PublicAPI
+class OutOfMemoryError(RayError):
+    """Indicates that the node is running out of memory and is close to full.
+
+    This is raised if the node is low on memory and tasks or actors are being evicted to free up memory.
+    """
+
+    def __str__(self, error_message):
+        if error_message:
+            message = "\n" + error_message
+        else:
+            message = "\nNode is running low on memory"
```

Comment:
Can we improve this error message? It should include information similar to the message that we print on the raylet.

Reply:
Also, these should inherit from RayTaskError.